src/Control/Effect/Parse.hs

```diff
@@ -7,9 +7,13 @@ module Control.Effect.Parse
 , parserForBlob
 , parseWith
 , parsePairWith
+  -- * Re-exports
+, Algebra
+, Has
+, run
```

Comment:
The idiom is now for effect modules to re-export `Algebra`, `Has`, and `run`, to reduce the number of imports required by clients of these interfaces.